packages/pluggableWidgets/datagrid-date-filter-web/src/components/FilterComponent.tsx

```diff
@@ -22,12 +24,21 @@ interface FilterComponentProps {
     screenReaderInputCaption?: string;
     tabIndex?: number;
     styles?: CSSProperties;
-    updateFilters?: (value: Date | null, type: DefaultFilterEnum) => void;
+    updateFilters?: (
+        value: Date | null,
+        rangeValues: [Date | null, Date | null] | null,
+        type: DefaultFilterEnum
+    ) => void;
 }
 
 export function FilterComponent(props: FilterComponentProps): ReactElement {
     const [type, setType] = useState<DefaultFilterEnum>(props.defaultFilter);
     const [value, setValue] = useState<Date | null>(null);
+    const [rangeValues, setRangeValues] = useState<[Date | null, Date | null] | null>(
+        props.defaultStartDate || props.defaultEndDate
+            ? [props.defaultStartDate ?? null, props.defaultEndDate ?? null]
+            : null
```

Comment:
Is there a particular use case for having `null` here instead of `[]` like in different parts of this PR? At those places it's also mainly just `?? []`, so I don't immediately see if `null` is strictly necessary.

Reply:
Fixed